server/routes/shared_api/place.py

```diff
@@ -853,17 +747,15 @@ def api_ranking_chart(dcid):
     place_type = "Country"
   else:
     place_type = get_place_type(dcid)
-    parent_place_list = parent_places(dcid).get(dcid, [])
+    parent_place_list = parent_places([dcid]).get(dcid, [])
     for parent in parent_place_list:
-      parent_type_list = parent.get("types", [])
       parent_place_dcid = parent.get("dcid", "")
-      if parent_type_list:
-        parent_type = parent_type_list[0]
-        # All wanted place types plus continent except CensusZipCodeTabulationArea.
-        if parent_type == "Continent" or (
-            parent_type in ALL_WANTED_PLACE_TYPES and
-            parent_type != "CensusZipCodeTabulationArea"):
-          break
+      parent_type = parent.get('type', '')
```

Comment:
super nit: only this line as different quote style, be consistent?

Reply:
fixed